### PR TITLE
Add main class

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

**Comment by jdfreder** [comment]
@rgbkrk @rgbkrk @rgbkrk @rgbkrk

File: src/transformime.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            if (bundle.data && renderer.mimetype in bundle.data) {
+                richRenderer = renderer;
+            }
+        }
+
+        if (bundle.data){
+            let data = bundle.data[richRenderer.mimetype];
+            element = richRenderer.render(data, metadata);
+            return element;
+        }
```
Comment (rgbkrk) [inline src/transformime.js]: Should we return a list of elements here instead?

Then we can let them reason about which to display, externally.

Also, it seems like we should use renderMimetype so we're not duplicating code here.

**Comment by rgbkrk** [comment]
I can go ahead and merge this then bring in the missing renderer classes as well.

File: src/transformime.js (hunk 1 above)
Comment (jdfreder) [inline src/transformime.js]: Won't that take a lot of memory?

**Pull request merged**